src/mwoffliner.lib.ts

```diff
@@ -325,6 +325,9 @@ async function execute(argv: any) {
     // await trimUnmirroredPages(downloader); // TODO: improve simplify graph to remove the need for a second trim
   }
 
+  const date = new Date();
+  const contentDate = `${date.getFullYear()}-${(`0${date.getMonth() + 1}`).slice(-2)}`;
```

Comment:
Something like `const filenameDate = (new Date()).toISOString().slice(0, 7)` seems better to me. Only one line and simplier.

Reply:
`(new Date()).toISOString().slice(0, 7)`
 will give "2020-06", which is only year and month. Just to confirm we do want the date as well, right? If yes, I just need to change the slice param.